bot/exts/info/pypi.py

```diff
@@ -32,6 +36,11 @@ async def get_package_info(self, ctx: Context, package: str) -> None:
         )
         embed.set_thumbnail(url=PYPI_ICON)
 
+        if (character := re.search(ILLEGAL_CHARACTERS, package)) is not None:
+            embed.description = f"Illegal character passed into command: '{escape_markdown(character.group(0))}'"
```

Comment:
Missing a `.` full stop at the end of the sentence,

```suggestion
            embed.description = f"Illegal character passed into command: '{escape_markdown(character.group(0))}'."
```

Reply:
I tested this a while ago. I didn't think it looked right. Here's a comparison.

With period:
![image](https://user-images.githubusercontent.com/15021300/110239644-db3d0600-7efc-11eb-8eb5-9cc8648f3863.png)

And without:
![image](https://user-images.githubusercontent.com/15021300/110239602-b0eb4880-7efc-11eb-9b26-b8b73a56abe2.png)

Reply:
Yeah, even I had the same feeling.